recipes/glslang/all/conanfile.py

```diff
@@ -138,13 +150,27 @@ def package_info(self):
         self.cpp_info.components["oglcompiler"].names["cmake_find_package"] = "OGLCompiler"
         self.cpp_info.components["oglcompiler"].names["cmake_find_package_multi"] = "OGLCompiler"
         self.cpp_info.components["oglcompiler"].libs = ["OGLCompiler" + lib_suffix]
+
+        if tools.Version(self.version) >= "11.5.0":
+            # GenericCodeGen
+            self.cpp_info.components["genericcodegen"].names["cmake_find_package"] = "GenericCodeGen"
+            self.cpp_info.components["genericcodegen"].names["cmake_find_package_multi"] = "GenericCodeGen"
+            self.cpp_info.components["genericcodegen"].libs = ["GenericCodeGen" + lib_suffix]
+
+            # MachineIndependent
+            self.cpp_info.components["machineindependent"].names["cmake_find_package"] = "MachineIndependent"
+            self.cpp_info.components["machineindependent"].names["cmake_find_package_multi"] = "MachineIndependent"
+            self.cpp_info.components["machineindependent"].libs = ["MachineIndependent" + lib_suffix]
```

Comment:
Shouldn't it also depend on glslang-core?
self.cpp_info.components["spirv"].requires = ["glslang-core"]

Reply:
It's glslang that depends on MachineIndependent, if you have a look at glslang/CMakeLists.txt you'll find:

target_link_libraries(glslang PRIVATE OGLCompiler OSDependent MachineIndependent)

It looks like the tests are not linking to the right libraries for the GCC build.